packages/siop-oid4vp/lib/rp/RP.ts

```diff
@@ -225,6 +227,19 @@ export class RP {
     return this._verifyResponseOptions
   }
 
+  public getResponseRedirectUri(mappings?: Record<string, string>): string | undefined {
+    if (this._responseRedirectUri === undefined) {
```

Comment:
Why are you constantly explicitly checking against undefined. That can have undesired side effects if someone explicitly sets something to null for instance (which is a valid value for an optional const/var/param). Simply do `!param`